src/Vlc.DotNet.Core/VlcMediaPlayer/VlcMediaPlayer.cs

```diff
@@ -149,8 +152,8 @@ public void Play()
         /// <param name="options">The options to be given</param>
         public void Play(FileInfo file, params string[] options)
         {
-            this.SetMedia(file, options);
-            this.Play();
+            SetMedia(file, options);
```

Comment:
It's a cosmetic change. Can these be excluded from the PR?

Reply:
Ok. 
Would you prefer a different PR for cosmetics?
I kind of like the use of "this." but to keep consistency about the rest of the code I felt removing those were harmless.

Reply:
I don't think this is high priority. I also like `this.` but we don't have clear coding styles.

Reply:
It won't take much time to pass some automatic tools over the code to clean some cosmetic warnings, like:
- Using directive is not required by the code and can be safely removed
- Redundant use of "this.". Although we could argue about modern coding styles and personal preferences, but at least I would enforce the code to be consistent and having sometimes "this" and sometimes nothing is confusing.

Not sure about other warnings regarding C# 6.0 features and so on. But at least the first 2 I mention takes a few seconds to fix.